SparkPost/2web2ui
# AC-1286: Tabs for join page

### What Changed
 - Add tabs for the join page

### How To Test
 - Log out and go to join
 - Modify JoinPage code by modifying `const brand` to `CROSS_LINK_MAP['spc']` or `CROSS_LINK_MAP['spceu']`
 - Verify the link navigates to the appropriate app.
